### ISSUE TITLE
NVDA + Firefox does not read updated text in aria-labelledby referenced element

### ISSUE THREAD

**Issue by n-chardon** [nvaccess/nvda#9387]
When labelling a button using the aria-labelledby attribute, using Firefox, the label read by NVDA is not updated when the content of the element referenced by the aria-labelledby is updated. 

### Steps to reproduce:
Open the following CodePen : https://codepen.io/n-chardon/pen/wOxQbW 
Focus the first Button : its label should read "New label for the button".
Click the "Update button text" button

### Actual behavior:
On Firefox, NVDA keeps reading the previous name of the button : "New label for the button"

### Expected behavior:
NVDA should read  "Updated text". 
On Chrome, NVDA behaves as expected.

#### NVDA installed/portable/running from source:
installed

#### NVDA version:
2018.4
#### Windows version:
 Windows 10 Enterprise N version 1803 (10.0.17134 N )

#### Name and version of other software in use when reproducing the issue:
Mozilla Firefox 

#### Other information about your system:
System Language FR
#### Does the issue still occur after restarting your PC?
Yes

#### Have you tried any other versions of NVDA?
No

**Comment by DrSooom** [comment]
@n-chardon: Please update Firefox to 66.0 and try again. CC: @jcsteh and @MarcoZehe

**Comment by jcsteh** [comment]
This is a Firefox bug: https://bugzilla.mozilla.org/show_bug.cgi?id=493683

**Comment by n-chardon** [comment]
This bug is 10 years old.  Are ou sure it is still relevant?

**Comment by jcsteh** [comment]
Yes, it is still relevant.

**Comment by jcsteh** [comment]
Fixed in Firefox 87 (currently Nightly).
